libraries/AP_HAL_ChibiOS/hwdef/include/minimize_common.inc

```diff
@@ -0,0 +1,107 @@
+# for now we make this define, but we should migrate from this to much
+#  more specific defines - added below.
+define HAL_MINIMIZE_FEATURES 1
+
+# disable emitting nice strings when activating RC channel aux functions:
+define AP_RC_CHANNEL_AUX_FUNCTION_STRINGS_ENABLED 0
+
+# low-end boards aren't expected to be used in environments where
+# things like satellite communications are required:
+define HAL_HIGH_LATENCY2_ENABLED 0
+
+# Gripper isn't a vital feature for smaller boards
+define AP_GRIPPER_ENABLED 0
+
+# Sprayer isn't a vital feature for smaller boards
+define HAL_SPRAYER_ENABLED 0
+
+# RunCam control isn't available on smaller boards
```

Comment:
you've missed some of my changes here - minimize_common should be the superset and then the other two a subset. e.g. RUNCAM

Reply:
Fixed.